### PR TITLE
doc: FORM Generation example

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: src/main/docs/guide/views/fieldset/fieldsetExample.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+NOTE: The field types and the validation annotations in the previous code sample influence the form generation.
+
+Thanks to the form generation API, you could create a controller such as:
+
+[source,java]
+----
+include::{testssuitefieldsetthymeleaf}/java/io/micronaut/views/fields/thymeleaf/BookController.java[tag=clazz]
+----
+
+https://micronaut.io/launch?features=views-thymeleaf[Micronaut Launch] or the Micronaut Command Line Interface (CLI) generate Thymeleaf fragments to render a form when you select the `views-thymeleaf` feature.
```
Comment (sdelamo) [inline src/main/docs/guide/views/fieldset/fieldsetExample.adoc:34]: I will create a PR in starter for these dependency and the templates

File: src/main/docs/guide/views/fieldset/fieldsetExample.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+[source, html]
+----
+<form action="/books/save" method="post">
+    <div class="mb-3">
+        <label for="title" class="form-label">Title</label>
+        <input type="text" name="title" value="" id="title" class="form-control" required="required"/>
+    </div>
+    <div class="mb-3">
+        <label for="pages" class="form-label">Pages</label>
+        <input type="number" name="pages" value="" id="pages" class="form-control" required="required"/>
```
Comment (timyates) [inline src/main/docs/guide/views/fieldset/fieldsetExample.adoc:12]: Why is there no `min` in this tag?  The BookSave field is annotated with `@Min(1)` and the inputnumber.html template has `th:min="${el.min()}"` but the generated source has no `min` attribute 🤔

**Review by timyates — APPROVED** [review]

File: src/main/docs/guide/views/fieldset/fieldsetExample.adoc (hunk 2 above)
Comment (sdelamo) [reply src/main/docs/guide/views/fieldset/fieldsetExample.adoc:12]: because, we don't generate those yet.

Comment (timyates) [reply src/main/docs/guide/views/fieldset/fieldsetExample.adoc:12]: Ahhh 😁

**Pull request merged**